starter_code/elevator.js

```diff
@@ -3,17 +3,68 @@ class Elevator {
     this.floor      = 0;
     this.MAXFLOOR   = 10;
     this.requests   = [];
+    this.direction = "up";
+    this.interval;
+    this.waitingList = [];
+    this.passengers = [];
   }
+  
+  start() { 
+    this.interval = setInterval(() => {
```

Comment:
This.floorUp y This.floorDown irian dentro de this.update. No dentro del setInterval()